code_schemes/rqas/aik/aik_political_participation.json

```diff
@@ -37,11 +37,115 @@
             "StringValue": "maybe",
             "VisibleInCoda": true
         },
+        {
+            "CodeID": "code-e3a5cbac",
+            "CodeType": "Normal",
+            "DisplayText": "yes, civic duty or democratic right",
+            "NumericValue": 4,
+            "StringValue": "yes_civic_duty_or_democratic_right",
+            "VisibleInCoda": true
+        },
+        {
+            "CodeID": "code-2e3ab664",
+            "CodeType": "Normal",
+            "DisplayText": "yes, assured of peace or security",
+            "NumericValue": 5,
+            "StringValue": "yes_assured_of_peace_or_security",
+            "VisibleInCoda": true
+        },
+        {
+            "CodeID": "code-6146d21c",
+            "CodeType": "Normal",
+            "DisplayText": "yes, to change leadership",
+            "NumericValue": 6,
+            "StringValue": "yes_to_change_leadership",
+            "VisibleInCoda": true
+        },
+        {
+            "CodeID": "code-412e8a89",
+            "CodeType": "Normal",
+            "DisplayText": "no, distrust for politics or politicians",
+            "NumericValue": 7,
+            "StringValue": "no_distrust_for_politics_or_politicians",
+            "VisibleInCoda": true
+        },
+        {
+            "CodeID": "code-06df1701",
+            "CodeType": "Normal",
+            "DisplayText": "no, busy at work",
+            "NumericValue": 8,
+            "StringValue": "no_busy_at_work",
+            "VisibleInCoda": true
+        },
+        {
+            "CodeID": "code-8eb58c57",
+            "CodeType": "Normal",
+            "DisplayText": "no, health constraints",
+            "NumericValue": 9,
+            "StringValue": "no_health_constraints",
+            "VisibleInCoda": true
+        },
+        {
+            "CodeID": "code-2cf5e03c",
+            "CodeType": "Normal",
+            "DisplayText": "no, fear of insecurity or violence",
+            "NumericValue": 10,
+            "StringValue": "no_fear_of_insecurity_or_violence",
+            "VisibleInCoda": true
+        },
+        {
+            "CodeID": "code-066ef3e1",
+            "CodeType": "Normal",
+            "DisplayText": "no or not interested",
+            "NumericValue": 11,
+            "StringValue": "no_or_not_interested",
+            "VisibleInCoda": true
+        },
+        {
+            "CodeID": "code-df40be57",
+            "CodeType": "Normal",
+            "DisplayText": "yes, love for politics",
```

Comment:
Is it worth sorting these yes, no, don't know?

Reply:
Updated, thanks.